config.default.edn

```diff
@@ -90,6 +90,7 @@
                                                          :pyreclimate "https://climate.pyregence.org/geoserver"}}
 
  ;; workers (server)
+ ;;TODO this needs to be updated with weather station pull, will time configuration go here?
```

Comment:
There's a TODO here obviously, but just flagging it. To answer your question, no the time configuration won't go here. `:pyregence.weather-stations/get-observation-stations-every-n-minutes 1440` is the right place for it.